core/src/allocation.jl

```diff
@@ -1,30 +1,47 @@
+"""
+Temporary solution before 'edge_ids_flow_inv' is no longer needed.
```

Comment:
This is a temporary fix, will be removed after refactoring allocation with the metagraph